lib/puppet/settings/environment_conf.rb

```diff
@@ -66,13 +66,11 @@ def manifest
     if disable_per_environment_manifest
       environment_conf_manifest = absolute(raw_setting(:manifest))
       if environment_conf_manifest && fallback_manifest_directory != environment_conf_manifest
-        errmsg = ["The 'disable_per_environment_manifest' setting is true, but the",
-        "environment located at #{@path_to_env} has a manifest setting in its",
-        "environment.conf of '#{environment_conf_manifest}' which does not match",
-        "the default_manifest setting '#{puppet_conf_manifest}'. If this",
-        "environment is expecting to find modules in",
-        "'#{environment_conf_manifest}', they will not be available!"]
-        Puppet.err(errmsg.join(' '))
+        #TRANSLATORS 'disable_per_environment_manifest' is a setting and 'environment.conf' is a file name and should not be translated
+        message = _("The 'disable_per_environment_manifest' setting is true, but the environment located at %{path_to_env} has a manifest setting in its environment.conf of '%{environment_conf}' which does not match the default_manifest setting '%{puppet_conf}'.") %
```

Comment:
Is this a message that we'd want to break up into multiple translate sections so that it would cost less if we change it in the future? As is, it's a bit difficult to read on one long line

Reply:
Sentence fragments are hard to translate into other languages. So we're suppose to be making them single sentences to be translated. If its really too long we'd have to break it into multiple sentences.

Reply:
That makes a lot of sense